modules/500-upmeter/images/smoke-mini/werf.inc.yaml

```diff
@@ -0,0 +1,34 @@
+---
+artifact: {{ .ModuleName }}/build-smoke-mini-artifact
+from: {{ .Images.BASE_GOLANG_16_ALPINE }}
```

Comment:
please use higher version of golang, to prevent CVE's appearing

Reply:
done